ckanext/dia/views.py

```diff
@@ -121,3 +127,27 @@ def edit_uri(uri_id):
             vars['error_summary'] = { 'Error': _('An unknown error occurred') }
 
     return tk.render('uris/edit.html', extra_vars=vars)
+
+
+api = Blueprint(u'dia-api', __name__, url_prefix=u'/dia-api')
+
+@api.route('/datapusher-health', methods=['GET', 'POST'])
+def datapusher_health():
+    a_day_ago = datetime.datetime.now() - datetime.timedelta(days=1)
+
+    query = model.Session.query(model.TaskStatus)\
+        .filter(_and_(
+            model.TaskStatus.state == 'error',
+            model.TaskStatus.task_type == 'datapusher',
+            model.TaskStatus.last_updated > a_day_ago
+        ))
```

Comment:
Returns any datapusher tasks that have errored in the last day

Reply:
I assume we don't have to worry about timezone issues here

Reply:
I don't think so... server runs in UTC I believe, and all the last_updated values appear to be set in that timezone also.